packages/modules/viewer-react/src/components/BaseViewer.tsx

```diff
@@ -107,6 +114,11 @@ export const BaseViewer: React.FC<ViewerProps> = ({
   // TODO not signed in message / loader
   return (
     <ErrorBoundary>
+      {!authorized && (
+        <div className="not-signed-in-container">
```

Comment:
Could you look at using the following component?
import { FillCentered } from "@bentley/ui-core";

Wonder how this will look and if its easier to use.